### PR TITLE
[ID-494]BBs Api for getting group memberships

### LINKED ISSUE #494: [FEATURE] BBs Api for getting group memberships
**Is your feature request related to a problem? Please describe.**
BBs API that gets the group membership name and status by accountID
[
   {"name":"groups-name-1", "status":"groups-status1"},
   {"name":"groups-name-2", "status":"groups-status2"}
]

### PR DESCRIPTION
**Pull request by stefanvit**
## Description
BBs Api for getting group memberships

**Resolves #494 
## Review Time Estimate
- [ ] Immediately
- [ ] Within a week
- [x] When possible

## Type of changes
- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
- [x] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [x] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [x] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by stefanvit** [comment]
Hi @petyos i've tested the API without permissions and i think it works fine. I want to ask you, should i use one of the other permission or should i make a new for this bbs API.

File: driven/storage/database.go [hunk 1]
```diff
@@ -403,7 +403,7 @@ func (m *database) applyGroupsChecks(groups *collectionWrapper) error {
 		}
 	}
 
-	name := "title_unique"
+	/*name := "title_unique"
```
Comment (petyos) [inline driven/storage/database.go:406]: Why was this commented?

File: driver/web/rest/bbapis.go [hunk 2]
```diff
@@ -49,3 +49,32 @@ func (h *BBSApisHandler) GetEventUserIDs(log *logs.Log, req *http.Request, user
 
 	return log.HTTPResponseSuccessJSON(data)
 }
+
+// GetGroupMemberships Gets all related group memberships status and group title using userID
+// @Description  Gets all related group memberships status and group title using userID
```
Comment (petyos) [inline driver/web/rest/bbapis.go:54]: Do not put the old way of docs approach.

File: driven/storage/adapter_events.go [hunk 3]
```diff
@@ -233,3 +233,62 @@ func (sa *Adapter) FindEventUserIDs(context TransactionContext, eventID string)
...
+	if err != nil {
+		return nil, err
+	}
+
+	return groups, nil
+}
+
+// FindGroupMembershipStatusAndGroupTitle Find all group membership status and gtoupTitle
+func (sa *Adapter) FindGroupMembershipStatusAndGroupTitle(context TransactionContext, userID string) ([]model.GetGroupMembershipsResponse, error) {
+	groupMembership, err := sa.FindGroupMembershipsByUserID(userID)
```
Comment (petyos) [inline driven/storage/adapter_events.go:266]: Do not make two calles to the database but implement only one call using join.

**Review by petyos — CHANGES_REQUESTED** [review]
Please take into account the feedback.

**Review by petyos — APPROVED** [review]
LGTM

**Pull request merged**